metricbeat/module/kubernetes/container/container.go

```diff
@@ -20,7 +20,6 @@ package container
 import (
 	"fmt"
 
-	"github.com/elastic/beats/v7/libbeat/logp"
```

Comment:
Why we delete it here?

Reply:
the linter was complaining that the logger variable was never used. With that gone, we didn't need to import this library anymore